shared/common-adapters/proof-broken-banner.tsx

```diff
@@ -20,9 +20,13 @@ type ProofBrokenBannerNonEmptyProps = {
 const ProofBrokenBannerNonEmpty = (props: ProofBrokenBannerNonEmptyProps) => {
   const dispatch = Container.useDispatch()
   const onClickUsername = React.useCallback(
-    isMobile
-      ? (username: string) => dispatch(ProfileGen.createShowUserProfile({username}))
-      : (username: string) => dispatch(Tracker2Gen.createShowUser({asTracker: true, username})),
+    (username: string) => {
+      if (isMobile) {
+        dispatch(ProfileGen.createShowUserProfile({username}))
+      } else {
+        dispatch(Tracker2Gen.createShowUser({asTracker: true, username}))
+      }
+    },
```

Comment:
Interesting -- isn't the original marginally faster?

Reply:
this type of optimization makes absolutely no difference. this way the typing is easier for ts